packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.tsx

```diff
@@ -13,10 +13,15 @@ const WalletsListingRoute: React.FC = () => {
     const firstLoginid = walletAccounts?.[0]?.loginid;
 
     useEffect(() => {
-        if (!activeWallet && firstLoginid) {
-            switchAccount(firstLoginid);
+        if (!activeWallet) {
+            const activeLinkedAccount = walletAccounts?.find(wallet => wallet.is_linked_account_active);
+            if (activeLinkedAccount) {
+                switchAccount(activeLinkedAccount.loginid);
```

Comment:
for future thinking (I know it wouldn't be viable to fix this anytime soon), 
what I realised recently, is that every single component can and do switchAccount at any given time, 
meaning, we have lots of race hazards, conflicts and complete unpredicatbility when whole application at random switches active account)
would be really good to establish strategy for that, as current one, is gonna generate more and more problems (as it already does in some areas), 
cc @markwylde-deriv